### PR TITLE
Tournament: ensure points are sorted numerically not lexically

### PR DESCRIPTION
**Pull request by glennj**
Issue #1801

### CODE REVIEW

File: exercises/tournament/canonical-data.json [hunk 1]
```diff
@@ -191,6 +191,25 @@
...
+      "uuid": "f9e20931-8a65-442a-81f6-503c0205b17a",
+      "description": "ensure points sorted numerically",
+      "property": "tally",
+      "input": {
+        "rows": [
+          "A;B;win",
+          "A;B;win",
+          "A;B;win",
+          "A;B;win",
+          "A;B;loss"
```
Comment (ErikSchierboom) [inline exercises/tournament/canonical-data.json:205]: All the other test cases use made-up names for the teams. Maybe we should be consistent and use them here too?

Comment (SleeplessByte) [reply exercises/tournament/canonical-data.json:205]: That be a nice change, yes.

**Comment by glennj** [comment]
Who is another appropriate reviewer?

**Comment by SleeplessByte** [comment]
Yes, and @petertseng is great too!

**Review by kotp — APPROVED** [review]

**Pull request merged**